Lib/cookielib.py

```diff
@@ -266,7 +270,7 @@ def http2time(text):
     return _str2time(day, mon, yr, hr, min, sec, tz)
 
 ISO_DATE_RE = re.compile(
-    """^
+    r"""^
```

Comment:
Is there a reason that this `r` is needed specifically for python 2? I suppose it is preferred, but gives the same regex with or without (`str(sre_parse.parse("""^...` with and without give the same result).
Unrelated to this change, I just noticed that the backslash in `[-\/]` doesn't do anything. Not sure why it's there.

Reply:
Oh I see the `r` wasn't in the python2 branch.

Reply:
So.... adding the ``r`` is correct, right?

Reply:
yes. Ignore me.